scrapers/github.js

```diff
@@ -0,0 +1,97 @@
+var request = require('sync-request');
+
+const USER_AGENT = "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 " + 
+    "(KHTML, like Gecko) Chrome/58.0.3029.81 Safari/537.36";
+const REQUEST_HEADER = {
+    headers: {"User-Agent": USER_AGENT}
+};
+const ENCODING = "UTF-8";
+const GITHUB_USERS_URL = "https://api.github.com/search/users";
+
+function searchGithubProfiles(query) {
+    var requestUrl = GITHUB_USERS_URL + "?q=" + query;
+    var res = request("GET", requestUrl, REQUEST_HEADER);
+    return JSON.parse(res.body.toString(ENCODING));
+}
+
+function createUserData(searchUserData) {
+    var dataItem = {
+        user_name: searchUserData.login,
+        user_id: searchUserData.id,
+        avatar_url: searchUserData.avatar_url,
+        atom_feed_link: searchUserData.html_url + ".atom"
+    };
+
+    // user's followers
+    var followersData = getExtraUserInfo(searchUserData.followers_url);
+    dataItem.followers = followersData.length;
+    dataItem.followers_data = followersData;
+    // user following
```

Comment:
Just to make Sure to be consistent and easy readability of code please insert a newline after each set.